frontend/src/component/project/Project/ProjectFeatureToggles/ProjectFeatureToggles.tsx

```diff
@@ -505,92 +512,118 @@ export const ProjectFeatureToggles = ({
         <>
             <PageContent
                 isLoading={loading}
+                disablePadding
```

Comment:
Most of the rest of the changes in this file is due to this one line pushing the content down